source/_integrations/smartenergy_goecharger.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: go-e Charger Cloud
+description: Instructions on setting up go-e Charger within Home Assistant.
+ha_category:
+  - Car
+  - Sensor
+  - Button
+ha_iot_class: Cloud Polling
+ha_quality_scale: silver
+ha_release: '2023.2'
+ha_config_flow: true
+ha_codeowners:
+  - '@openkfw'
+ha_domain: smartenergy_goecharger
+ha_platforms:
+  - button
+  - number
+  - select
+  - sensor
+ha_integration_type: hub
+---
+
+The go-e Charger Cloud integration allows you to control and monitor the go-e wallbox and car connected to it. Integration currently support following features:
+
+- connect to the wallbox via cloud API and retrieve the status
+- portion of the retrieved status is displayed via sensors
+- there is a button included to control charging and authentication based on the car state
+- there is a number input included to change charging power in Amperes
+- there is a select input included to change the phase (1 phase or 3 phase)
+- button, number, and select controls are disabled if the wallbox is offline
+- plethora of services are available to be used by other integrations or in automations
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+Following sensors are available to monitor the wallbox (and car respectively):
+
+| Name                         | Description                                                      |
+| ---------------------------- | ---------------------------------------------------------------- |
+| `car_status`                 | State of the car - connected/charging/etc.                       |
```

Comment:
This seems like an entity name? Although, those aren't guaranteed. It might be better to use human names for the individual sensors instead.

Reply:
Adjusted a bit.